query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -45,6 +44,7 @@ enum Inner {
 /// Holding the information to reconnect the engine if needed.
 #[derive(Debug, Clone)]
 struct EngineDatamodel {
+    #[allow(unused)]
```

Comment:
You should be able to remove this. If the client tests fail then we can look at adding it back.